### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hpar12**
Just created a template for the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,18 @@
-# coaching-tutorial
+# Project Title
+
+One Paragraph of project description goes here
+
+### Prerequisites
+
+### Running the Application
+
+### Tesing the Application
```
Comment (fraserc) [inline README.md]: Spelling ;)

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
-# coaching-tutorial
+# Project Title
+
+One Paragraph of project description goes here
+
+### Prerequisites
+
+### Running the Application
+
```
Comment (fraserc) [inline README.md]: Okay, so first up there's some code already here so how about how do I compile it and how do I run it? This is a good place to start as it'll get you thinking about the different stages in the build.

File: README.md (hunk 1 above)
Comment (fraserc) [inline README.md]: And same here, from the command line how would I run the tests in here?

Comment (hpar12) [reply README.md]: Updated spelling

**Pull request closed without merging**